iis-wf/iis-wf-affmatching/src/test/java/eu/dnetlib/iis/wf/affmatching/match/voter/AffOrgMatchVotersFactoryTest.java

```diff
@@ -113,8 +122,10 @@ public void createSectionedNameLevenshteinCountryLooseMatchVoter() {
         assertInternalVotersCount(voter, 2);
         
         assertTrue(getInternalVoter(voter, 0) instanceof CountryCodeLooseMatchVoter);
-        assertTrue(getInternalVoter(voter, 1) instanceof SectionedNameLevenshteinMatchVoter);
-        
+        AffOrgMatchVoter voter1 = getInternalVoter(voter, 1);
+        assertTrue(voter1 instanceof SectionedNameLevenshteinMatchVoter);
+        assertTrue(Whitebox.getInternalState(voter1, "getOrgNamesFunction") instanceof GetOrgNameFunction);
```

Comment:
I can see three newly added lines (above) which are repeated 5 times in different test methods which are quite similar (only classes verified with `instanceof` are different) . Maybe we could make parameterized `assert` method out of that?